imap/jmap_calendar.c

```diff
@@ -8108,6 +8109,19 @@ static int jmap_calendarevent_participantreply(struct jmap_req *req)
         goto done;
     }
 
+    /* Compare partStat */
+    param = icalproperty_get_first_parameter(attendee, ICAL_PARTSTAT_PARAMETER);
```

Comment:
Calling `icalproperty_get_first_parameter` with a NULL attendee won't cause a crash, but still is considered an error by libical, e.g. it will set libical's errno to `ICAL_BADARG_ERROR`. I think this whole conditional block could be moved up in the loop that sets that `attendee`, right before the `break`.

Reply:
Good call. Fixed